plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/notification/NotificationPluginRegistry.java

```diff
@@ -47,6 +47,10 @@ public void removePluginInterests(String pluginId) {
         }
     }
 
+    public boolean isAnyPluginInterestedIn(String notificationName) {
+        return !getPluginsInterestedIn(notificationName).isEmpty();
+    }
+
     public Set<String> getPluginsInterestedIn(String notificationName) {
```

Comment:
`isAnyPluginInterestedIn` is just doing `!notificationNameToPluginsInterestedMap.get('plugin-id').isEmpty()`. I don think it will cause concurrent modification exception. why do you think that would be the case? Do you think `notificationNameToPluginsInterestedMap` should be `ConcurrentHashMap` instead of just the `HashMap`?